ets-demo/etsdemo/tests/test_main.py

```diff
@@ -35,27 +35,30 @@ def fake_configure_traits(instance):
 
 
 def mock_demo_launch():
-    """ Mock Demo.configure_traits so that we can open and then close the UI
+    """Mock Demo.configure_traits so that we can open and then close the UI
     for testing the main function.
     """
     return mock.patch.object(Demo, "configure_traits", fake_configure_traits)
 
 
 class TestMain(unittest.TestCase):
-    """ Test main function."""
+    """Test main function."""
 
     @require_gui
     def test_main(self):
         # Main function must be launchable even if there are no data available.
         # In normal running situation, no loggings should be emitted
         argv = ["etsdemo"]
         mocked_io = io.StringIO()
-        with mock_iter_entry_points({}), \
-                mock_demo_launch(), \
-                mocked_io, \
-                mock.patch("sys.stdout", mocked_io), \
-                mock.patch("sys.stderr", mocked_io), \
-                mock.patch("sys.argv", argv):
+        with mock_iter_entry_points(
+            {}
+        ), mock_demo_launch(), mocked_io, mock.patch(
+            "sys.stdout", mocked_io
+        ), mock.patch(
+            "sys.stderr", mocked_io
+        ), mock.patch(
+            "sys.argv", argv
+        ):
```

Comment:
sigh. these look awkward. but we can't use `(...)` in `with` statements in Python 3.6 IIUC. Not much we can do if we can to keep the codebase black-happy.